src/styles/navbar.css

```diff
@@ -0,0 +1,53 @@
+.navbar {
+  width: 100%;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
```

Comment:
- [ ] Your Project is working fine. Please adjust the "padding" left and right because your design need to look like the original design
- Your design
-
![y1](https://user-images.githubusercontent.com/63972670/168151392-2e086eb7-9d51-4282-a26d-115abba661d7.PNG)
- Original. 
-
![y2](https://user-images.githubusercontent.com/63972670/168151616-e5d95b84-8a5f-41af-be7d-29f496354574.PNG)

Reply:
Thank you, I did that.